include/phasar/PhasarLLVM/DataFlowSolver/Mono/Solver/MonoCache.h

```diff
@@ -0,0 +1,64 @@
+/******************************************************************************
+ * Copyright (c) 2020 Philipp Schubert.
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of LICENSE.txt.
+ *
+ * Contributors:
+ *     Ajay Subramanya Kudli Prasanna Kumar, Philipp Schubert and others
+ *****************************************************************************/
+
+#ifndef PHASAR_PHASARLLVM_MONO_SOLVER_MONOCACHE_H_
+#define PHASAR_PHASARLLVM_MONO_SOLVER_MONOCACHE_H_
+
+#include<iostream>
+#include<unordered_map>
+#include<vector>
+
+namespace psr {
+  template <class K, class V>
+class MonoCache {
+  // TODO
+public:
+        unordered_map<K, V> summaries;
+        vector<int> time; //The most recently added or recently used key is placed last
+        int c;
+
+        MonoCache(int capacity) {
+                c = capacity;
+        }
+
+void get(K key) {
+        unordered_map<K, V>::iterator iter = summaries.find(key);
+        if(iter == summaries.end())
+        return -1; //not found
```

Comment:
This function returns `void`.

Reply:
changed the return type to template variable K